packages/dx-grid-core/src/plugins/local-sorting/computeds.js

```diff
@@ -31,5 +30,40 @@ export const sortedRows = (rows, sorting, getCellValue, getColumnCompare) => {
       () => 0,
     );
 
+const sortTree = (tree, compare, groupCompare) => {
+  if (!tree.length) return tree;
+  const sorted = mergeSort(tree, tree[0].groupRow ? groupCompare : compare);
+  sorted.forEach((row) => {
+    if (row.items && row.items.length) {
+      const sortedItems = sortTree(row.items, compare, groupCompare);
+      row.items.splice(0, row.items.length, ...sortedItems);
+    }
+  });
+
+  return sorted;
+};
+
+export const sortedRows = (
+  rows,
+  sorting,
+  getCellValue,
+  getColumnCompare,
+  isGroupRow,
+  getRowLevelKey,
+) => {
+  if (!sorting.length) return rows;
+  const compare = createCompare(sorting, getColumnCompare, getCellValue);
+
+  if (isGroupRow) {
```

Comment:
Why do you check for the isGroupRow Getter presence? Seems like tree sorting should be applied when tree structure defined by the getRowLevelKey Getter.